osidb/tests/test_package_versions.py

```diff
@@ -11,30 +11,29 @@ class TestPackageVersions(object):
     def test_create_cve_v5_version(self):
         """test raw verison range creation"""
 
-        vr_1 = CVEv5Version.objects.create(
-            version="3.2.1",
+        flaw = FlawFactory()
+        pkg = PackageFactory(
+            acl_read=flaw.acl_read,
+            acl_write=flaw.acl_write,
+            flaw=flaw,  # make _validate_acl_identical_to_parent_flaw happy
```

Comment:
`pkg = PackageFactory(flaw=flaw)` should be sufficient.

Reply:
Fixed in commit [c20061e](https://github.com/RedHatProductSecurity/osidb/pull/307/commits/c20061e53092a9add1dd0a35563d31d53e07849e)